docker-compose.test.yml

```diff
@@ -7,10 +7,11 @@ services:
       - TOOLPAD_DATABASE_URL=postgresql://postgres:postgres@postgres:5432/postgres?connect_timeout=10
       - TOOLPAD_ENABLE_CREATE_BY_DOM=1
       - PORT=3000
-      - EXTERNAL_URL=http://localhost:3000/
-      - TOOLPAD_EXTERNAL_URL=http://172.17.0.1:3000/
+      - TOOLPAD_EXTERNAL_URL=http://toolpad:3000/
```

Comment:
We'll run the playwright container from the same compose file, so both the toolpad container and playwright container are in the same network. So both on server and client `http://toolpad:3000/` will resolve